requirements/base.in

```diff
@@ -12,5 +12,6 @@ lazy==1.1
 requests==2.22.0
 pypng
 simplejson
+web-fragments
```

Comment:
whats a purpose of this ? add some explanation

Reply:
DeprecationWarning: xblock.fragment is deprecated. Please use web_fragments.fragment instead